CRUD - Django/README.md

```diff
@@ -0,0 +1 @@
+## https://www.javatpoint.com/django-crud-application
```

Comment:
I guess you let this link here for further reading, right?
When I add things in the code with the intention of doing something with it later (even if it's a simple thing that I think will be easy to remember), I try to be as much descriptive as I can in order to not forget what I wanted to do with it even if I keep the repository frozen for ages.
What do you think about adding some description like this?
```suggestion
## Django CRUD
🛠️  `under construction` 🛠️ 
This project's objective is to implement an application following this tutorial:
https://www.javatpoint.com/django-crud-application
```
If you have more ideas about _what_ you want the application to do, or something else you want to try with this projetc, add here as well so the Helena from the future can easily follow the steps you want as Helena from the present 😬